recipes/protobuf/all/test_package/CMakeLists.txt

```diff
@@ -6,11 +6,6 @@ conan_basic_setup(TARGETS)
 
 find_package(protobuf CONFIG REQUIRED)
 
-find_program(PROTOC_PROGRAM protoc)
```

Comment:
I'm okay with removing this because warnings are not failing the build, and I think people just skip them

Reply:
It seems too weak to be a test as well!

Reply:
The build will fail if not found :)
I guess:
```cmake
if (NOT PROTOC_PROGRAM)
    message(FATAL_ERROR "Protoc was not found")
endif()
```
could make sense though.

But having `find_program(PROTOC_PROGRAM protoc)` here is just confusing since that is already covered by `find_package(protobuf`. It got me confused at least.